src/Common/AsynchronousMetrics.cpp

```diff
@@ -985,6 +986,8 @@ void AsynchronousMetrics::update(TimePoint update_time, bool force_update)
     }
 #endif
 
+    new_values["TrackedMemory"] = { total_memory_tracker.get(), "Memory tracked by ClickHouse (should be equal to MemoryTracking metric), in bytes." };
```

Comment:
Maybe we should add the new metric to docs (https://clickhouse.com/docs/operations/system-tables/asynchronous_metrics), what do you think ?

Reply:
I would prefer not to, since I don't want people to use it, it should be equal to `MemoryTracking` from `system.metrics`. The reason that this metric exposed is to debug memory tracker leaks, there is a comment in the commit - https://github.com/ClickHouse/ClickHouse/pull/83755/commits/7ac369e168d1e8dc230ce0c6cfe2ef188909b75a